### DISCUSSION TITLE
Custom location for $ACME_DIR for a CA ?

### DISCUSSION THREAD

**Discussion by positiwise** [acmesh-official/acme.sh#4724] [Q&A]
Hello,

Just wanted to ask if there is any way to have custom location for $ACME_DIR or some other way I can override validation path where the challenge will be updated ? I am asking because I am working on supporting a well known CA (through custom ACME that I am writing), but their validation challenge file path is like http://[domain.name]/.well-known/pki-validation/fileauth.txt instead of usual ACME : /.well-known/acme-challenge/  . Is there any way I can indicate that from custom ACME Server to acme.sh to go through custom file validation path?

**Comment by neilpang** [comment]
you can define "ACME_DIR=xxxxx" in the `account.conf` file.
but I don't think it can help in your case.
